modules/tables/table_34_Harsh.html

```diff
@@ -0,0 +1,197 @@
+
+<div class="container">
+	
+	<div class="table">
+		<div class="table-header">
+			<div class="header__item"><a id="name" class="filter__link" href="#">Name</a></div>
+			<div class="header__item"><a id="wins" class="filter__link filter__link--number" href="#">Wins</a></div>
+			<div class="header__item"><a id="draws" class="filter__link filter__link--number" href="#">Draws</a></div>
+			<div class="header__item"><a id="losses" class="filter__link filter__link--number" href="#">Losses</a></div>
+			<div class="header__item"><a id="total" class="filter__link filter__link--number" href="#">Total</a></div>
+		</div>
+		<div class="table-content">	
+			<div class="table-row">		
+				<div class="table-data">Tom</div>
+				<div class="table-data">2</div>
+				<div class="table-data">0</div>
+				<div class="table-data">1</div>
+				<div class="table-data">5</div>
+			</div>
+			<div class="table-row">
+				<div class="table-data">Dick</div>
+				<div class="table-data">1</div>
+				<div class="table-data">1</div>
+				<div class="table-data">2</div>
+				<div class="table-data">3</div>
+			</div>
+			<div class="table-row">
+				<div class="table-data">Harry</div>
+				<div class="table-data">0</div>
+				<div class="table-data">2</div>
+				<div class="table-data">2</div>
+				<div class="table-data">2</div>
+			</div>
+		</div>	
+	</div>
+</div>
+
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700');
+
+$base-spacing-unit: 24px;
+$half-spacing-unit: $base-spacing-unit / 2;
+
+$color-alpha: #1772FF;
+$color-form-highlight: #EEEEEE;
+
+*, *:before, *:after {
+	box-sizing:border-box;
+}
+
+body {
```

Comment:
Use precise class names so that your module does not affect another module or outer elements using this tag. Using "body" would affect the entire page... using h1, h2 , table, try, etc would affect other h1 tags or tables with the page. Therefore, Use classes.

Reply:
done

Reply:
Also , please add hacktoberfest and hacktobest-accepted label . Thanks